### PR TITLE
Test io cube

### PR DESCRIPTION
**Pull request by bayerl**
Implement basic tests of cube file inread functions

### CODE REVIEW

File: canela/bin/soap.py [hunk 1]
```diff
@@ -87,7 +89,9 @@ def create_soap(filepath, nmax, lmax, rcut, procs):
     print('ALL DONE')
 
 
-# if __name__ == '__main__':
-#     create_soap()
 def main():
-    create_soap()
+    # print version if that is only arg
+    if len(sys.argv) > 1 and sys.argv[1] == '--version':
+        print('soap, ' + __version__)
```
Comment (michael-cowan) [inline canela/bin/soap.py]: I used this approach for argparse, but there's an easier way with click:
You can add the following decorator to your script function:
@click.version_option(\_\_version\_\_)

File: canela/bin/soap.py [hunk 2]
```diff
@@ -90,8 +91,4 @@ def create_soap(filepath, nmax, lmax, rcut, procs):
 
 
 def main():
-    # print version if that is only arg
-    if len(sys.argv) > 1 and sys.argv[1] == '--version':
-        print('soap, ' + __version__)
-    else:
-        create_soap()
+    create_soap()
```
Comment (paulboone) [inline canela/bin/soap.py]: delete main();

**Comment by michael-cowan** [comment]
Everything looks good!

**Comment by paulboone** [comment]
Tests look great!

**Pull request merged**